src/main/java/frc/robot/systems/BallHandlingFSM.java

```diff
@@ -79,10 +84,10 @@ public BallHandlingFSM() {
 		pDH = new PowerDistribution(1, ModuleType.kRev);
 
 		// Reset state machine
-		pushCommandTimeStamp = Timer.getFPGATimestamp() - Constants.PUSH_TIME_SECONDS;
+		pushCommandTimeStamp = Timer.getFPGATimestamp() - Constants.TIME_FOR_FULL_SHOT;
 		reset();
 		handleRetractIntakeMechState(null);
-		handleRetractingState(null);
+		handleRetractShooterState(null);
```

Comment:
Shouldn't these be put inside the reset()? We want to do this any time we reset the FSM right?

Reply:
Once calling reset(), the system enters START_STATE, which retracts these systems in its handler, I am not totally sure why we are calling these handlers separately in the constructor, so I have removed those 2 useless lines